pbspro-rpmlintrc

```diff
@@ -1,4 +1,6 @@
 # IGNORE: The following are purposely suppressed.
+addFilter("non-executable-script")
+addFilter("explicit-lib-dependency")
```

Comment:
Please see https://pbspro.atlassian.net/wiki/spaces/PD/pages/794263558/Packaging+PBS+Pro+for+Third+Party+Linux+Repositories

We should be able to address this warning by removing the Requires: lines for libedit and libical in pbspro.spec and pbspro.spec.in

Reply:
Thanks @mike0042 ! Let me try your suggestion.